generate.sh

```diff
@@ -139,13 +140,32 @@ generateJavascript() {
   return 0
 }
 
+generatePython() {
+  LIBRARY="$1"
+  OPERATION="$2"
+  ARTIFACT_ID="symbol-openapi-$LIBRARY-client"
+  echo "Generating $LIBRARY"
+  rm -rf "$BUILD_DIR/$ARTIFACT_ID"
+  openapi-generator generate -g "$LIBRARY" \
+    -o "$BUILD_DIR/$ARTIFACT_ID" \
+    -t "$LIBRARY-templates/" \
+    -i "$INPUT" \
+    --additional-properties="projectName=$ARTIFACT_ID" \
+    --additional-properties="packageVersion=$VERSION" \
+    --additional-properties="snapshot=$SNAPSHOT" \
+    --type-mappings=x-number-string=int
```

Comment:
I'm not expert in python, is int good enough for unsigned long / BigInteger without overflowing? 

- Double-check if the client can parse unsigned string numbers like "1683298087010368300" from rest into 1683298087010368300 int number
- Double-check if the client can serialize int number like 1683298087010368300 into string number "1683298087010368300" when sending a payload. 

The x-number-string is mapper customization so the user doesn't need to convert from string to number every time there is a string number. Check if it works when creating int from string numbers.

Reply:
Yes, in fact **int** is the only kind of integer since Python 3.0. Python 2.x had _ints_ and _long ints_ which were unified in [PEP-237](https://www.python.org/dev/peps/pep-0237/) and fully implemented in Python 3.x. 

Parsing and serialising of very large int numbers were tested as part of catbuffer-generators for Python. 

Yes x-number-string mapper customisation works when creating int from string numbers.